client/app/visualizations/details/DetailsRenderer.jsx

```diff
@@ -1,17 +1,36 @@
 import React, { useState } from 'react';
-import map from 'lodash/map';
+import { map, mapValues, keyBy } from 'lodash';
+import moment from 'moment';
 import { RendererPropTypes } from '@/visualizations';
+import { clientConfig } from '@/services/auth';
 import Pagination from 'antd/lib/pagination';
 
 import './details.less';
 
+function renderValue(value, type) {
+  const formats = {
+    date: clientConfig.dateFormat,
+    datetime: clientConfig.dateTimeFormat,
+  };
+
+  if (type === 'date' || type === 'datetime') {
+    if (moment.isMoment(value)) {
```

Comment:
I think `if (moment.isMoment(value)) {` is enough, previous `if` is almost useless.

Reply:
It's relevant to make sure we have a known format type. But it's true that it will only be a moment if it's one of those (in normal situations).